controllers/replicationdestination_controller.go

```diff
@@ -93,11 +96,12 @@ func (r *ReplicationDestinationReconciler) Reconcile(req ctrl.Request) (ctrl.Res
 	// Only reconcile if the replication method is internal
 	if inst.Spec.Rsync != nil {
 		result, err = RunRsyncDestReconciler(ctx, inst, r, logger)
+	} else if inst.Spec.Rclone != nil {
```

Comment:
a hierarchy has been defined here with Rsync taking precedence over Rclone if both are set. Is there code that prevents both from being defined? If not, is this hierarchy documented?

Reply:
It's not a hierarchy, it's "either" or "or". It is documented that both rsync and rclone can be used as an option for DR.

Reply:
but if `Spec.Rsync is not nil` then the Rclone test is skipped so, at least to me, that indicates that Rsync has precedence over Rclone, unless I'm missing something else that corrects for this. I admit to not reading all of the code...

Reply:
That's correct. It is a bug.

Reply:
@husky-parul can you file the bug for this?

Reply:
I will fix in this PR itself

Reply:
you rock!

Reply:
Actually, it is not a bug. For a particular instance of `Replication*` it can either be `rclone` or `rsync` not both. 
@cooktheryan

Reply:
My point was not that this is a bug but rather that a hierarchy between rclone and rsync has been established in the code. This may be perfectly fine but it should be acknowledged somewhere (docs, readme), or it should be disallowed for the user to specify both.

Reply:
Added #80 for this